types/three/three-core.d.ts

```diff
@@ -5389,17 +5389,72 @@ export class Skeleton {
     clone(): this;
 }
 
+/**
+ * A mesh that has a Skeleton with bones that can then be used to animate the vertices of the geometry. The material must support skinning and have skinning enabled.
+ * @see MeshStandardMaterial.skinning
+ * @source https://github.com/mrdoob/three.js/blob/master/src/objects/SkinnedMesh.js
+ */
 export class SkinnedMesh extends Mesh {
-    constructor(geometry?: Geometry | BufferGeometry, material?: Material | Material[], useVertexTexture?: boolean);
 
+    /**
+     * @param  {BufferGeometry} geometry? an instance of {@link BufferGeometry}.
+     * @param  {Material|Material[]} material? (optional) an instance of {@link Material}. Default is a new {@link MeshBasicMaterial}.
+     * @param  {boolean} useVertexTexture?
+     */
+    constructor(geometry?: BufferGeometry, material?: Material | Material[]);
+
+    /**
+     * Either "attached" or "detached". "attached" uses the SkinnedMesh.matrixWorld property for the base transform	matrix of the bones. "detached" uses the SkinnedMesh.bindMatrix. Default is "attached".
+     */
     bindMode: string;
+
+    /**
+     * The base matrix that is used for the bound bone transforms.
+     */
     bindMatrix: Matrix4;
+
+    /**
+     * The base matrix that is used for resetting the bound bone transforms.
+     */
     bindMatrixInverse: Matrix4;
+
+    /**
+     * Used to check whether this or derived classes are skinned meshes. Default is true.
+     * You should not change this, as it used internally for optimisation.
+     */
+    isSkinnedMesh: boolean;
```

Comment:
I'm not sure it's a good idea to add this to the .d.ts file, given their guidance to not change it.
I think it would be better to leave this off so that consumers of the library don't accidentally think it's ok to change, but if they do need to change it, they can use a cast and document why they need to change it. What do you think?

Reply:
If you don´t add that property, the compiler will complain because the library checks that property.